### ISSUE TITLE
The serialization of UserParameter has changed and doesn't work with the UI

### ISSUE THREAD

**Issue by schaubl** [slipstream/SlipStreamServer#353]
The server started to serialize `UserParameter` like the following:
```xml
<entry>
    <string>Exoscale.username</string>
    <userParameter category="Exoscale" description="Exoscale account username" mandatory="true" name="Exoscale.username" order="10" order_="10" readonly="false" type="RestrictedString">
        <value/>
    </userParameter>
</entry>
```

instead of: 
```xml
<entry>
    <string>Exoscale.username</string>
    <parameter category="Exoscale" description="Exoscale account username" mandatory="true" name="Exoscale.username" order="10" order_="10" readonly="false" type="RestrictedString">
        <value/>
    </parameter>
</entry>
```

`<parmeter>` is now `<UserParameter>`.
Because of that, the UI is not able to see any UserParameter.

**Comment by schaubl** [comment]
I deployed a new SlipStream from `snapshot` and it had this issue. 
So this has to be fixed before the release.

**Comment by schaubl** [comment]
I haven't found a reason in the code for this change.

I think the reason is the update of `simple-xml` from `2.2.4` to `2.7.1`.
See https://github.com/slipstream/SlipStream/blame/master/pom.xml#L503.

So in this case the UI will have to be updated.

**Comment by schaubl** [comment]
I confirm that the cause is a side effect of the migration from `simple-xml 2.2.4` to `simple-xml 2.7.1`.

**Comment by mebster** [comment]
This is easy to fix by specifying the name the element should have using @ElementList or @ElementArray attribute.

**Comment by schaubl** [comment]
@mebster : It's was already fixed here: #354.
I wasn't able to fix it with a parameter of @ElementMap.
